Dockerfile

```diff
@@ -10,20 +10,30 @@ RUN case ${TARGETPLATFORM} in \
     "darwin/arm64")  TARGET="aarch64-apple-darwin" ;; \
     "linux/amd64")   TARGET="x86_64-unknown-linux-musl" ;; \
     "linux/arm64")   TARGET="aarch64-unknown-linux-musl" ;; \
-    "windows/amd64") TARGET="x86_64-pc-windows-gnu" ;; \
     *) \
     echo "ERROR: TARGETPLATFORM '${TARGETPLATFORM}' not supported." \
     exit 1 \
     ;; \
     esac &&\
-    nix --accept-flake-config --extra-experimental-features 'nix-command flakes' build -L ".#wasmcloud-${TARGET}" &&\
-    install -Dp ./result/bin/wasmcloud /out/wasmcloud
+    nix --accept-flake-config --extra-experimental-features 'nix-command flakes' build -L ".#wasmcloud-${TARGET}"
+RUN install -Dp ./result/bin/wash /out/wash
+RUN install -Dp ./result/bin/wasmcloud /out/wasmcloud
 
 FROM debian:${DEBIAN_VERSION}-slim as result
 
 RUN apt update &&\
     apt install -y ca-certificates
 
-COPY --from=build /out/wasmcloud /bin/wasmcloud
+ARG USERNAME=wasmcloud
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+RUN addgroup --gid $USER_GID $USERNAME \
+    && adduser --disabled-login -u $USER_UID --ingroup $USERNAME $USERNAME
+
+USER $USERNAME
+
+COPY --from=build --chown=$USERNAME --chmod=755 /out/wash /bin/wash
+COPY --from=build --chown=$USERNAME --chmod=755 /out/wasmcloud /bin/wasmcloud
```

Comment:
We may want to reconsider including the wash binary inside of the wasmcloud image, since it changes the size + ability of the container

Reply:
makes sense, happy to remove in a follow-up

Reply:
Oh, wait, this particular change only concerns a development build actually, we never use this Dockerfile to build out own images we release. So this is actually fine I think, but we should remove the `wash` binary from our released images

Reply:
Can be addressed in a followup